db/migrate/20190406135546_add_phone_numbers_to_surveys.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhoneNumbersToSurveys < ActiveRecord::Migration[5.2]
+  def change
+    add_column :surveys, :phone_numbers, :text
```

Comment:
Now that we are storing the phone numbers we should add functionality somewhere to make sure that the phone numbers are valid